### PR TITLE
1.2 feature/user dash data

### PR DESCRIPTION
**Pull request by danbergelt**
# Description

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [X] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] Test A
- [ ] Test B

# Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

**Review by BDesselle — DISMISSED** [review]

**Review by dionys93 — DISMISSED** [review]

File: mission-control-client/src/actions/activeProjectActions.js [hunk 1]
```diff
@@ -1,7 +1,24 @@
...
 export const SET_ACTIVE_PROJECT = "SET_ACTIVE_PROJECT";
+export const SET_ACTIVE_PROJECT_SUCCESS = "SET_ACTIVE_PROJECT_SUCCESS";
+export const SET_ACTIVE_PROJECT_FAILURE = "SET_ACTIVE_PROJECT_FAILURE";
 
 export const setActiveProject = el => {
   return dispatch => {
-    dispatch({type: SET_ACTIVE_PROJECT, payload: el})
-  }
-}
+    dispatch({ type: SET_ACTIVE_PROJECT });
```
Comment (agordon1123) [inline mission-control-client/src/actions/activeProjectActions.js:10]: We could do SET_ACTIVE_PROJECT_START when we do a code cleanup to avoid confusion

**Review by agordon1123 — DISMISSED** [review]

**Review by agordon1123 — APPROVED** [review]

**Review by BDesselle — APPROVED** [review]

File: mission-control-client/src/components/dashboard/user-dashboard/DashboardBanner.js [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
 
 const DashboardBanner = props => {
+
   const programs = ["web", "ux/ui", "ds"];
 
   return (
     <div className="dashboard-banner-container">
       <div className="dashboard-banner-head">
-        <p>{props.activeProductStore.active ? props.activeProductStore.active.name : "Loading..."}</p>
+        <p>{props.activeProjectStore.active ? props.activeProjectStore.active.project.project.name : "Loading..."}</p>
```
Comment (dionys93) [inline mission-control-client/src/components/dashboard/user-dashboard/DashboardBanner.js]: `project.project` is questionable. Why can't it be just `props.activeProductStore.active.name`?

Comment (danbergelt) [reply mission-control-client/src/components/dashboard/user-dashboard/DashboardBanner.js]: In our code cleanup we can for sure change this

File: mission-control-client/src/components/dashboard/user-dashboard/DashboardBanner.js [hunk 3]
```diff
@@ -29,8 +30,12 @@ const DashboardBanner = props => {
           )}
         </div>
         <div className="dashboard-product-status">
-          <p>In Progress</p>
-        </div> 
+          {props.activeProjectStore.active && new Date(props.activeProjectStore.active.project.project.end) > new Date() ? (
```
Comment (dionys93) [inline mission-control-client/src/components/dashboard/user-dashboard/DashboardBanner.js]: `project.projectDetails`

**Review by dionys93 — CHANGES_REQUESTED** [review]
`project.projectDetails`

**Review by dionys93 — APPROVED** [review]

**Pull request merged**